org.libretro.RetroArch.json

```diff
@@ -49,13 +49,13 @@
         }
       ],
       "post-install": [
-        "mkdir -p /app/share/icons/hicolor/scalable/apps/",
-        "mv /app/share/pixmaps/retroarch.svg /app/share/icons/hicolor/scalable/apps/",
-        "rmdir /app/share/pixmaps/",
-        "mkdir -p /app/etc",
-        "cp retroarch.cfg /app/etc",
-        "mkdir -p /app/share/appdata",
-        "cp org.libretro.RetroArch.appdata.xml /app/share/appdata"
+        "mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/",
+        "mv ${FLATPAK_DEST}/share/pixmaps/retroarch.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/",
+        "rmdir ${FLATPAK_DEST}/share/pixmaps/",
+        "mkdir -p ${FLATPAK_DEST}/etc",
+        "sed s:@prefix@:${FLATPAK_DEST}:g retroarch.cfg > ${FLATPAK_DEST}/etc/retroarch.cfg",
```

Comment:
This is some fancy stuff right here. Nicely done :+1:

Reply:
It would be much better if RetroArch allowed setting the defaults at build time. Maybe there actually is a such possibility?

Reply:
Great idea. Made an issue over at https://github.com/libretro/RetroArch/issues/8637 . Compiled and tested, going to merge this one in the mean time.